src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -739,10 +739,8 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                 if (provider._codeFixService != null &&
                     supportsFeatureService.SupportsCodeFixes(document))
                 {
-                    var result = await Task.Run(
-                        () => provider._codeFixService.GetMostSevereFixableDiagnostic(
-                            document, range.Span.ToTextSpan(), cancellationToken),
-                            cancellationToken).ConfigureAwait(false);
+                    var result = await provider._codeFixService.GetMostSevereFixableDiagnostic(
+                            document, range.Span.ToTextSpan(), cancellationToken).ConfigureAwait(false);
```

Comment:
there is only 1 caller. and that caller calls this method under Task.Run already.